<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterJoueur</class>
 <widget class="QDialog" name="InterJoueur">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="bout_annul">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bout_ech">
     <property name="text">
      <string>Proposer un échange</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bout_mess">
     <property name="text">
      <string>Envoyer un message privé</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bout_def">
     <property name="text">
      <string>Défier</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
